recipe/meta.yaml

```diff
@@ -45,14 +43,18 @@ test:
     - nuitka.tools
     - nuitka.tree
     - nuitka.utils
-  requires:
-    - pip
+  #requires:
+  #  - pip
```

Comment:
I think you could remove the commented code (but leave the comment explaining reasoning)